### PR TITLE
DDT-1267 moved the parseSort into the Shared-Kernel

### PR DESCRIPTION
**Pull request by pimg**
resolves: DDT-1267

### CODE REVIEW

**Review by douradofrazer — APPROVED** [review]

File: shared-kernel-infrastructure/src/main/java/org/dcsa/skernel/infrastructure/sorting/Sorter.java [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+import org.springframework.data.domain.Sort;
+import org.springframework.stereotype.Component;
+
+import java.util.Arrays;
+import java.util.List;
+import java.util.Set;
+
+/** Helper class for transforming the DCSA Sorting syntax to Spring-data Sorting. */
+@Slf4j
+@Component
```
Comment (nt-as) [inline shared-kernel-infrastructure/src/main/java/org/dcsa/skernel/infrastructure/sorting/Sorter.java:15]: Does it make sense for this to be a `@Component` when we cannot use bean injection anyway?

File: shared-kernel-infrastructure/src/main/java/org/dcsa/skernel/infrastructure/sorting/Sorter.java [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+import org.dcsa.skernel.infrastructure.pagination.Cursor;
+import org.springframework.data.domain.Sort;
+import org.springframework.stereotype.Component;
+
+import java.util.Arrays;
+import java.util.List;
+import java.util.Set;
+
+/** Helper class for transforming the DCSA Sorting syntax to Spring-data Sorting. */
+@Slf4j
```
Comment (nt-as) [inline shared-kernel-infrastructure/src/main/java/org/dcsa/skernel/infrastructure/sorting/Sorter.java:14]: We do not seem to be using the logging, so the annotation looks like it is unused.

File: shared-kernel-infrastructure/src/main/java/org/dcsa/skernel/infrastructure/sorting/Sorter.java (hunk 1 above)
Comment (pimg) [reply shared-kernel-infrastructure/src/main/java/org/dcsa/skernel/infrastructure/sorting/Sorter.java:15]: component removed

File: shared-kernel-infrastructure/src/main/java/org/dcsa/skernel/infrastructure/sorting/Sorter.java (hunk 2 above)
Comment (pimg) [reply shared-kernel-infrastructure/src/main/java/org/dcsa/skernel/infrastructure/sorting/Sorter.java:14]: annotation removed

**Review by nt-as — APPROVED** [review]

**Pull request merged**